src/index.ts

```diff
@@ -7,46 +7,46 @@
  * without explicit written authorization from Numenta Inc.
  */
 import * as core from "@actions/core";
+import { DynamoDBClient, GetItemCommand, GetItemInput } from "@aws-sdk/client-dynamodb";
+import { fromEnv } from "@aws-sdk/credential-providers";
+import { marshall } from "@aws-sdk/util-dynamodb";
 import { Octokit } from "octokit";
 
 const CLA_LINK = "https://github.com/thousandbrainsproject/cla/issues/new?template=01_sign_cla.yml&labels=signature+CLA+v1&title=Contributor+License+Agreement+v1+Signature";
+const CLA_SIGNATORIES_TABLE_NAME = "DynamoDB-CLASignatoriesTable-12GCQ0F0TB473";
 
-const TBP_BOT_TOKEN_SECRET = process.env.TBP_BOT_TOKEN_SECRET;
 
-if (!TBP_BOT_TOKEN_SECRET)
-{
-    core.setFailed("TBP_BOT_TOKEN_SECRET is not set.");
-    process.exit(1);
-}
+const prAuthor = core.getInput("pull-request-author");
 
-const tbpBotOctokit = new Octokit(
+const dynamodb = new DynamoDBClient(
     {
-        auth: TBP_BOT_TOKEN_SECRET
+        credentials: fromEnv(),
+        region: "us-east-2"
```

Comment:
non-blocking:
I think this should be an environment variable for the same reasons as the previous comment.

Reply:
I agree that would make things more portable. I'll defer as future work for now.